pkg/langruntime/langruntime.go

```diff
@@ -298,6 +301,11 @@ func (l *Langruntimes) GetCompilationContainer(runtime, funcName string, install
 		command = fmt.Sprintf(
 			"sed 's/<<FUNCTION>>/%s/g' $GOPATH/src/controller/kubeless.tpl.go > $GOPATH/src/controller/kubeless.go && "+
 				"go build -o %s/server $GOPATH/src/controller/kubeless.go > /dev/termination-log 2>&1", funcName, installVolume.MountPath)
+	case strings.Contains(runtime, "java"):
+		command = "cp -r /usr/src/myapp/* /kubeless/ && " +
+			"cp /kubeless/*.java /kubeless/function/src/main/java/io/kubeless/ &&" +
+			"cp /kubeless/function-pom.xml /kubeless/function/pom.xml 2>/dev/null || true &&" +
+			"mvn package && mvn install"
```

Comment:
should we dump logs to `/dev/termination-log` ?

Reply:
Sure, will add, without which there is no insight into why function deployment failed.